src/lib/cli.ts

```diff
@@ -61,6 +62,41 @@ const setUpUserFeedback = (sonarInstance: Sonar, spinner: IORA) => {
     });
 };
 
+const confirmLaunchInit = () => {
+    debug(`Initiating launch init confirm.`);
+
+    const question: Array<object> = [{
+        message: `Configuration file doesn't exist or error in loading. Do you want to create a new config file?`,
```

Comment:
Maybe we can have something like `Unable to find a valid configuration file` or something similar instead of `doesn't exist or error` .

What do you thing?